src/storage/lock_manager/waiter_manager.rs

```diff
@@ -90,6 +94,12 @@ impl WaitTable {
         self.wait_table.iter().map(|(_, v)| v.len()).sum()
     }
 
+    fn sync_wait_table_is_empty(&self) {
```

Comment:
fn sync_wait_table_status(is_empty: bool) is better.

Reply:
This function only set `WAIT_TABLE_IS_EMPTY` to ture.